RetailCoder.VBE/UI/CodeExplorer/Commands/AddComponentCommand.cs

```diff
@@ -30,14 +30,28 @@ public bool CanAddComponent(CodeExplorerItemViewModel parameter)
 
         public void AddComponent(CodeExplorerItemViewModel node, ComponentType type)
         {
-            var components = node != null
+            using (var components = node != null
                 ? GetDeclaration(node).Project.VBComponents
-                : _vbe.ActiveVBProject.VBComponents;
+                : ComponentsColectionFromActiveProject())
```

Comment:
Typo - probably should be `ComponentsCollectionFromActiveProject`

Reply:
No typo; it is the Com collection `VBComponents` from the active project. 
This is only a separate function, because you cannot have anonymous functions in a ternary, at least AFAIK.

Reply:
Sorry, was kind of blind reading the comment. Yes, the `l` was missing.